src/main/resources/static/restaurantchoice.js

```diff
@@ -3,7 +3,7 @@
  */
      function myCreateFunction() {
        var table = document.getElementById("myTable");
-       var row = table.insertRow(0);
+       var row = table.insertRow(table.rows.length);

```

Comment:
I changed it so the new row will be added to the bottom of the table instead of above it